playwright/tests/e2e/features/pools.feature

```diff
@@ -0,0 +1,59 @@
+Feature: Check Pools Page Cardano Explorer
+
```

Comment:
There is multiple tests steps that make the same function like for example:
**Line 14:** Given the user is in the Pools details page of one Pool
**Line 19:** Given the user is in the Pools details site of one Pool
**Line 24:** Given the user is in the Pools details web of one Pool Comment
It is not necessary write different ways of the same step for each test indeed the idea is to reuse steps if is need it, so please take a look to those steps where logic is the same and just reuse them like:
**Line 14:** Given the user is in the Pools details page of one Pool
**Line 19:** Given the user is in the Pools details page of one Pool
**Line 24:** Given the user is in the Pools details page of one Pool